tests/system/providers/google/cloud/automl/resources/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Do we need ``__init__`` file in resources?

Reply:
@bhirsz this `__init__` file was added automatically by pre-commit script